src/deploy/functions/validate.ts

```diff
@@ -140,7 +140,7 @@ export async function secretsAreValid(projectId: string, wantBackend: backend.Ba
  * Ensures that all endpoints specifying secret environment variables target platform that supports the feature.
  */
 function validatePlatformTargets(endpoints: backend.Endpoint[]) {
-  const supportedPlatforms = ["gcfv1"];
+  const supportedPlatforms = ["gcfv1", "gcfv2"];
```

Comment:
Nit:

```
const supportedPlatforms = new Set(["gcfv1", "gcfv2"]);
const unsupported = endpoints.filter((e) => !supportedPlatforms.has(e.platform));
```

Nit x2: I don't know how often this is called, but maybe it makes more sense to have this defined as a const up top?

Reply:
You and @inlined should have a talk! @inlined does not like Sets, especially in our code where N is almost always < 50 where using Set (which has every so slightly different API from arrays) doesn't give us much!

Reply:
Team convention is king. :)

That said, my thoughts are its less about efficiency and more about intention/code clarity.

`Set` limits the purpose of the variable/code to just checking for existence.

`Array` leaves the implementation open ended. Maybe the array going to be used with `has`, `reduce`, or `map`. Someone else won't know unless they read through the code to find how/why the array is being used.